modelci/env-backend.env

```diff
@@ -1,3 +1,5 @@
-BACKEND_CORS_ORIGINS="http://localhost,http://localhost:3333,http://localhost:8080,https://localhost:3000,https://localhost:8080"
+BACKEND_CORS_ORIGINS=http://localhost,http://localhost:3333,http://localhost:8080,https://localhost:3000,https://localhost:8080,http://155.69.146.35:3333
```

Comment:
How do we avoid making this info public?

Is there a method to keep this secret and also provide easy to use?

Reply:
I am trying a bash script to generate such an environment file.

Reply:
I have already done and verify the code, will open another PR after this is done. I don't want to mess up this PR.